src/main/java/org/quantumbadger/redreader/fragments/MainMenuFragment.java

```diff
@@ -41,11 +42,30 @@
 import org.quantumbadger.redreader.views.liststatus.ErrorView;
 import org.quantumbadger.redreader.views.liststatus.LoadingView;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 import java.util.Collection;
 import java.util.HashSet;
 
 public class MainMenuFragment extends RRFragment implements MainMenuSelectionListener, RedditSubredditSubscriptionManager.SubredditSubscriptionStateChangeListener {
 
+	public static final int FRONTPAGE = 0;
```

Comment:
It would be good if these were prefixed with `MENU_MENU_ACTION_` (like most of the other IntDefs)